src/main/java/org/springframework/data/tarantool/core/query/TarantoolRepositoryQueryExecutor.java

```diff
@@ -7,5 +7,5 @@
  */
 public interface TarantoolRepositoryQueryExecutor {
 
-    Object execute(final Object[] parameters);
+    Object execute(Object[] parameters);
```

Comment:
Looks like we don't protect `parameters` against modification.
@ArtDu could you please look at this changes.

Reply:
@dkasimovskiy If I right understand your point, than it's not issue
https://stackoverflow.com/questions/5380177/final-arguments-in-interface-methods-whats-the-point